docker-compose.yml

```diff
@@ -91,7 +91,7 @@ services:
       LOGGING_LEVEL: "INFO"
       SPARQL_ENDPOINT: "http://db:8890/sparql"
       LIMIT_SIZE: "10000"
-      MU_AUTH_SUDO: "true"
+      MU_AUTH_SUDO: "false"
```

Comment:
Here I don't see why it would change thinking Is it possible that you'd need to merge dev into the hardsplit branch ? It look like some changes have not been picked up, like this one and the dump file publisher config ones haha